app/src/main/java/com/example/instagramclone/fragments/message_screen/MessageFragment.kt

```diff
@@ -0,0 +1,58 @@
+package com.example.instagramclone.fragments.message_screen
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.Navigation
+import androidx.recyclerview.widget.LinearLayoutManager
+import coil.load
+import coil.transform.CircleCropTransformation
+import com.example.instagramclone.adapters.MessageAdapter
+import com.example.instagramclone.adapters.MessageAdapter.urlUtil
+import com.example.instagramclone.databinding.FragmentMessageBinding
+import com.example.instagramclone.util.collectLatestLifecycleFlow
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class MessageFragment : Fragment() {
```

Comment:
Have we not created a composable for the Messages screen? If we have, then we can get rid of this Fragment and its layout file too.